docs/layers/arc-layer.md

```diff
@@ -130,7 +130,7 @@ Method called to retrieve the target position of each object.
 
 * Default: `[0, 0, 0, 255]`
 
-The rgba color at the source, in `r, g, b, [a]`. Each component is in the 0-255 range.
+The rgba color at the source, is `[r, g, b, [a]]`. `r, g, b, a` is in 0-255 range and `a` is 255 by default.
```

Comment:
`...in the format of [r, g, b, [a]]. Each channel is a number between 0-255...`

Reply:
@manishmz can you make this change and push?